packages/cdk-routing-split/test/setup/jest.setup.js

```diff
@@ -2,9 +2,7 @@
 const AWS = require('aws-sdk');
 
 AWS.config.update({ region: 'us-east-1' });
-if (process.env.DSI_PARAMETER_NAME && process.env.DSI_PARAMETER_NAME.includes('sandbox')) {
-    AWS.config.credentials = new AWS.SharedIniFileCredentials({ profile: 'sandbox' });
-}
+AWS.config.credentials = new AWS.SharedIniFileCredentials({ profile: 'sandbox' });
```

Comment:
this looks useful, whered you find this?

Reply:
@BennuAshes You copied here from paysafe project. I just deleted guard condition because I think it is not necessary, since this is temporary project. If platform move tests as well, then I think we would need to add DSI_PARAMETER_NAME to provider environment.

Reply:
@srdjan-drca the condition should remain. The sandbox credentials shouldn't be used in non-sandbox tests (the pipeline). The only thing that is temporary about this is how it's being deployed (Serverless-framework vs CDK). The project and tests are not temporary.

Reply:
@nthornton2010 How credentials are set for non-sandbox environment?

Reply:
this doesn't seem to impact the task so im going to merge this shortly